x-pack/docs/en/security/fips-140-compliance.asciidoc

```diff
@@ -147,9 +146,25 @@ for the file realm and the <<security-api-put-user,create users>> and
 <<security-api-change-password,change password>> APIs for the native and reserved realms.
 Other types of realms are not affected and do not require any changes.
 
+[discrete]
+==== Cached Password Hashing
+
+For cache hashing, salted `sha256` is recommended instead of `PBKDF2`. Though
+compliant with FIPS-140-2, `PBKDF2` is -- by design -- slow, and thus not
+generally suitable as a cache hashing algorithm. Cached credentials are never
+stored on disk, and salted `sha256` provides an adequate level of security for
+in-memory credential hashing, without imposing prohibitive performance overhead.
+You _may_ use `PBKDF2`, however you should carefully assess performance impact
+first. Depending on your deployment, the overhead of `PBKDF2` could undo most of
+the performance gain of using a cache.
+
+You should set the `cache.hash_algo` realm settings to `ssha256`. `ssha256` is
+the default value for all `cache.hash_algo` realm settings. See
```

Comment:
I am aware this reuses the existing wording, but `cache.hash_algo` is not only just realm settings, both service account and API key are not realms. So maybe just remove `realm`.